src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureNameCell/FeatureNameCell.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { Link, Typography } from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { useStyles } from './FeatureNameCell.styles';
+import { Highlighter } from 'component/common/Highlighter/Highlighter';
+import { useSearchHighlightContext } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+
+interface IFeatureNameCellProps {
+    name?: string;
+    project?: string;
+    description?: string;
+}
+
+export const FeatureNameCell: FC<IFeatureNameCellProps> = ({
+    name,
+    project,
+    description,
+}) => {
+    const { classes: styles } = useStyles();
+    const search = useSearchHighlightContext();
+
+    return (
+        <>
+            <Link
+                component={RouterLink}
+                to={`/projects/${project}/features/${name}`}
+                underline="hover"
+                data-loading
+            >
+                <ConditionallyRender
```

Comment:
Currently it's very hard to navigate to feature toggles with short feature names from this cell. We should find a better pattern. Going from the whole row being clickable to a few characters link might be a bit excessive.

<img width="1354" alt="Skjermbilde 2022-05-05 kl  12 10 24" src="https://user-images.githubusercontent.com/16081982/166904847-79e68848-7e29-4957-a212-00123324a59f.png">

Reply:
I can make entire `Feature toggle name` cell clickable, how about that?

Reply:
That would be a good improvement

Reply:
In next draft PR: https://github.com/Unleash/unleash-frontend/pull/951